src/pages/Offering.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react'
+
+import { PriceInfo } from '../sections/offering/price-info'
+import { InsuranceCoverage } from '../sections/offering/insurance-coverage'
+import { TopBar } from '../sections/offering/top-bar'
+import { InsuredAmount } from '../sections/offering/insured-amount'
+import styled from 'react-emotion'
+import { HedvigInfo } from '../sections/offering/hedvig-info'
+import { HedvigSwitch } from '../sections/offering/hedvig-switch'
+import { GetInsured } from '../sections/offering/get-insured'
+import { Legal } from '../sections/offering/legal'
+
+export const Container = styled('div')({})
+
+interface MyComponentState {
+  alreadyInsured: boolean
+}
+
+export class Offering extends React.Component<{}, MyComponentState> {
+  constructor(props: any) {
+    super(props)
+
+    this.setState({
+      alreadyInsured: false,
+    })
+
+    this.alreadyInsured = this.alreadyInsured.bind(this)
+  }
+
+  protected alreadyInsured() {
+    return this.state.alreadyInsured
```

Comment:
also i dont really see the point of this function? Why not access `this.state.alreadyInsured` directly?